app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="col-lg-3 mx-lg-auto m-2">
+<div class="card shadow">
+
+<h2 class="card-header text-center">Sign up</h2>
+
+<div class="card-body">
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+   <div class="form-group">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, class: "form-control" %>
+  </div>
+
+  <div class="form-group>
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "form-control" %>
+  </div>
+
+  <div class="form-group>
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
```

Comment:
- [ ] Great job working on the sign up with devise but it would be great if you made the styles the same given design since in the requirements you should stick with the design layout as much as possible so I could see in your design is different than the given design so please change it to be the same with the nav as shown ⬇🔥💯🤩

![image](https://user-images.githubusercontent.com/58553711/235317930-3833c764-8e50-402c-af3d-ba59ef826b74.png)



- Your
![image](https://user-images.githubusercontent.com/58553711/235317716-5fbc385f-4995-4503-aaa3-afd0fb324fc8.png)

- Given
![image](https://user-images.githubusercontent.com/58553711/235317746-ae07ac45-757d-41d8-a59a-31ae9feb52c3.png)